logic/src/commonMain/kotlin/com/wire/kalium/logic/sync/incremental/EventGatherer.kt

```diff
@@ -73,7 +77,8 @@ internal interface EventGatherer {
 internal class EventGathererImpl(
     private val eventRepository: EventRepository,
     private val incrementalSyncRepository: IncrementalSyncRepository,
-    private val slowSyncRepository: SlowSyncRepository
+    private val slowSyncRepository: SlowSyncRepository,
+    private val dispatchers: KaliumDispatcher = KaliumDispatcherImpl,
```

Comment:
Added `dispatchers` are not used anywhere, also the same with `slowSyncRepository` which is removed in similar PR for develop https://github.com/wireapp/kalium/pull/1656 so here it could also be removed, right?